docs/accessControl.md

```diff
@@ -108,6 +108,30 @@ The above example will give students 90 minutes for this exam, and they must sta
 
 **Note that time limits should not be set for exams in the CBTF (Computer-Based Testing Facility). Instead, such exams should set `"mode": "Exam"`, in which case `timeLimitMin` will have no effect and the time limits will be enforced by the CBTF scheduling software.**
 
+### Time limit adjustments for open assessments
+
+Changes to the time limits in the `infoAssessment.json` file only take effect before an assessment is started. Once a student opens an assessment and creates an instance, the time limit for that assessment will be set based on the time limit rules at the time the assessment started. In some scenarios, instructors or invigilators may need to change the time limit allotted to individual students or for all students with an open assessment instance. This change could be due to errors in the original configuration, unexpected events (e.g., power outages or scheduling problems), or concessions.
+
+On the Instructor View for the assessment, select the Students tab. This tab will show a list of all students that have started the assessment. The "Remaining" column will include one of the following values:
+
+* The value "Closed" indicates that the student has already finished the assessment, that their time ran out, or ([for exam assessments with `autoClose` set as `true`](assessment.md#auto-closing-exam-assessments)) that the student was inactive for over six hours. Students do not have the ability to submit answers for these assessments.
+* The value "Unlimited" indicates that the assessment is open and has no time limit (other than the `autoClose` setting listed above). Students are able to submit answers for these assessments.
+* A value indicating a number of minutes indicates the remaining time until the exam is automatically closed. Students are able to submit answers until this time expires. Note that this time is computed when the page is loaded, so to get an updated time, the page must be refreshed.
+
+To change the remaining time, or to close or re-open an assessment, click on the "Actions" button, which will open the following options:
```

Comment:
Good point, will do.

Reply:
Sorry, I thought you meant a different doc page, you meant the actual page. Since instructors won't be able to change the timeLimitMin there, I'm not sure how to best describe this option...

Reply:
I agree that it's a bit weird because you can't edit on the Access page. I just worry that people will think that editing an access rule to increase the time limit will cause that time limit to be extended for exams that are in progress. I think we saw this with some UBC exams in Fall, where instructors tried to extend time limits in this way.

I guess one possibility would be to actually support this, although it's a bit clunky. We wouldn't want to overwrite manual adjustments, so we'd probably need to store a flag like `time_limit_set_manually` so we would know which ones to extend on a sync. Personally I'm not very enthusiastic about doing this, as I think it would be confusing.

Maybe at the bottom of the Access page we could say "To change the access rules, [Edit `infoAssessment.json]" where we link to the editing page. And then underneath that we could say "Note that editing time limits in the access rules will not change the time limits for in-progress assessments. To change these, go to the [Students] tab."

Or maybe we just forget about all of this idea. Instructors will probably figure it out and we can always put a FAQ entry about it!

Reply:
Would a link to the access rules docs be an option?

Reply:
Absolutely, that's a good idea.